### DISCUSSION TITLE
How to Download New Base Models on 0.4.1

### DISCUSSION THREAD

**Discussion by anultravioletaurora** [open-webui/open-webui#7071] [Q&A] [answered]
Where can I go do download new base models like in 0.3.X? I don't see the option to download models from Ollama anymore in the Admin Settings, only the ability to reset or delete what I already have:

![image](https://github.com/user-attachments/assets/681eaef9-369b-4d53-bcca-6310b250dc76)

**Comment by flefevre** [comment, marked as answer]
Have a look here
https://github.com/open-webui/open-webui/discussions/7079
